scalableminds/webknossos
# Organization page

This PR adds an administration page to edit or delete an organization. 

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- log in as admin user
- the admin menu should contain the item "Organization"
- click on it
- You should be able to edit the display name and the new user email address. The current values should be the default values of the input fields. 
- the current pricing plan should be displayed
- on submit you should be redirected to the dataset list view and the organization's attributes should be updated
- on the edit page should also be a delete button
- on click there should pop up a confirm modal
- after confirming the action the organization should be deleted  

### Issues:
- fixes #5105

------
